### DISCUSSION TITLE
월드에서 패키지 만들때 에러나요

### DISCUSSION THREAD

**Discussion by ChanjongB** [naverz/zepeto-studio-kor#403] [5. Q&A]
빌드잇 작업한걸 월드에서 애니메이션 모델링 추가후 
패키지 만드는 작업이 실패해요 
제가 작업한게 아니라고 뜨네요

**Comment by ChanjongB** [comment]
>![에러0](https://user-images.githubusercontent.com/99932124/154601530-39a4104e-6532-4867-a882-eb898302c145.jpg)

**Reply by x4cf** [reply]
1.홈페이지에 등록한 월드 주소와 같은 주소인지 확인
2.월드를 등록한 아이디와 유니티 내에서 로그인된 아이디가 같은지 확인
을 해보시는게 좋을것같아요 :D

**Reply by ChanjongB** [reply]
아 완전감사합니다~
